src/components/Organisations/index.jsx

```diff
@@ -6,9 +6,12 @@ import organizationsData from '@site/static/generated/organisations.json'
 import styles from './styles.module.css'
 
 export default function Organisations({ maxItems, onlySponsors }) {
-  const orgs = onlySponsors //
-    ? organizationsData.filter((org) => org.sponsor)
-    : organizationsData
+  const orgs =
+    onlySponsors === 'true' //
+      ? organizationsData.filter((org) => org.sponsor)
+      : onlySponsors === 'false' //
+        ? organizationsData.filter((org) => !org.sponsor)
+        : organizationsData
```

Comment:
if onlySponsors is false we filter all sponsors?

Reply:
you are right - I have been lazy, will clean it

![image](https://github.com/user-attachments/assets/4c27432c-3d86-45ed-9daa-5da43ceb577c)

Reply:
Done @Uzlopak